webapp/graphite/render/utils.py

```diff
@@ -12,8 +13,14 @@ def extractPathExpression(tokens):
     elif tokens.pathExpression:
       pathExpressions.add(tokens.pathExpression)
     elif tokens.call:
-      for a in tokens.call.args:
-        extractPathExpression(a)
+      # if we're prefetching seriesByTag, look up the matching series and prefetch those
+      if tokens.call.funcname == 'seriesByTag':
+        if STORE.tagdb:
+          for series in STORE.tagdb.find_series([t.string[1:-1] for t in tokens.call.args if t.string]):
+            pathExpressions.add(series)
```

Comment:
maybe it would make sense to add a log statement if somebody submitted a tag query on a store that doesn't support tags.